app/components/works/admin_component.rb

```diff
@@ -19,7 +19,7 @@ def options
         ['Change owner', edit_owners_path(work)],
         ['Lock/Unlock', edit_locks_path(work)]
       ]
-      opts << ['Decommission', edit_decommission_path(work)] unless work.head.decommissioned?
+      opts << ['Decommission', edit_work_decommission_path(work)] unless work.head.decommissioned?
```

Comment:
Since both works and collections may be decommissioned now, name the path helpers more clearly.